server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -643,6 +643,34 @@ private void registerAsyncMetrics(MeterRegistry registry) {
             )
         );
 
+        metrics.add(
+            registry.registerLongAsyncCounter(
+                "es.indexing.low_watermark_splits.total",
+                "Total number of times bulk requests are split due to SPLIT_BULK_LOW_WATERMARK",
+                "operations",
+                () -> new LongWithAttributes(
+                    Optional.ofNullable(stats.getOrRefresh())
+                        .map(NodeStats::getIndexingPressureStats)
+                        .map(IndexingPressureStats::getLowWaterMarkSplits)
+                        .orElse(0L)
+                )
+            )
+        );
+
+        metrics.add(
+            registry.registerLongAsyncCounter(
+                "es.indexing.high_watermark_splits.total",
```

Comment:
Let's include `coordinating` in this name since it is at the coordinating level currently.

```
es.indexing.coordinating.low_watermark_splits.total
es.indexing.coordinating.high_watermark_splits.total
```